test/computer_board_test.rb

```diff
@@ -3,28 +3,23 @@
 require './lib/cell'
 require './lib/ship'
 require './lib/board'
-require './lib/computer'
+require './lib/computer_board'
 
 class ComputerBoardTest < Minitest::Test
 
   def setup
     @computer = ComputerBoard.new
+    @computer.computer_board.create_cells
   end
 
   def test_it_exists
     assert_instance_of ComputerBoard, @computer
   end
 
   def test_placement_of_ships
-    @computer.place_cruiser
+    @computer.place_ships
```

Comment:
do we create the ships here or in the game file based off the amount of ships the user chooses to create?

Reply:
here.